experimenter/experimenter/nimbus_ui_new/templates/nimbus_experiments/detail.html

```diff
@@ -1,14 +1,341 @@
+              <td colspan="3">{{ experiment.takeaways_summary|format_not_set|safe }}</td>
+            </tr>
+            <tr>
+              <th>Gain amount</th>
+              <td colspan="3">{{ experiment.takeaways_gain_amount|format_not_set |safe }}</td>
+            </tr>
+          </tbody>
+        </table>
+      </div>
+    </div>
+    <!-- Overview Card -->
+    <div class="card mb-3">
+      <div class="card-header">
+        <h4>Overview</h4>
+      </div>
+      <div class="card-body">
+        <table class="table table-striped">
+          <tbody>
+            <tr>
+              <th>Slug</th>
+              <td>{{ experiment.slug }}</td>
+              <th>Experiment owner</th>
+              <td>{{ experiment.owner }}</td>
+            </tr>
+            <tr>
+              <th>Application</th>
+              <td>{{ experiment.get_application_display }}</td>
+              <th>Public description</th>
+              <td>{{ experiment.public_description|format_not_set|safe }}</td>
+            </tr>
+            <tr>
+              <th>Feature config</th>
+              <td>
+                {% for feature in experiment.feature_configs.all %}
+                  {{ feature.name }}- {{ feature.description }}
+                {% empty %}
+                  <span style="color: red;">Not set</span>
+                {% endfor %}
+              </td>
+              <th>Advanced targeting</th>
+              <td>{{ experiment.targeting_config.name }}- {{ experiment.targeting_config.description }}</td>
+            </tr>
+            <tr>
+              <th>Hypothesis</th>
+              <td colspan="3">{{ experiment.hypothesis }}</td>
+            </tr>
+            <tr>
+              <th>Primary Outcomes</th>
+              <td colspan="3">
+                {% for outcome, url in primary_outcome_links %}
+                  <a href="{{ url }}" target="_blank" rel="noopener noreferrer">{{ outcome|format_to_title }}</a>
+                  {% if not forloop.last %},{% endif %}
+                {% empty %}
+                  <span style="color: red; ">Not set</span>
+                {% endfor %}
+              </td>
+            </tr>
+            <tr>
+              <th>Secondary Outcomes</th>
+              <td colspan="3">
+                {% for outcome, url in secondary_outcome_links %}
+                  <a href="{{ url }}" target="_blank" rel="noopener noreferrer">{{ outcome|format_to_title }}</a>
+                  {% if not forloop.last %},{% endif %}
+                {% empty %}
+                  <span style="color: red; ">Not set</span>
+                {% endfor %}
+              </td>
+            </tr>
+            <tr>
+              <th>Team projects</th>
+              <td>
+                {{ experiment.projects.all|join:"<br>"|format_not_set|safe }}
+              </td>
+              <th>Subscribers</th>
+              <td>
+                {{ experiment.subscribers.all|join:"<br>"|format_not_set|safe }}
```

Comment:
This is *not* safe. If there was HTML in the project names or subscriber names, this would inject it.

Reply:
Oh yeah, you don't need to use safestring here, just loop over them and format explicitly like

```
{% for subscriber in experiment.subscribers.all %}
  <p>{{ subscriber }}</p>
{% endfor %}
```

Funy I didn't even know you could do string joints like that in a template.

Reply:
me neither